pnpm-workspace.yaml

```diff
@@ -3,7 +3,9 @@ packages:
   - "docs"
   - "cli"
   - "packages/*"
-  - "turborepo-tests/*"
+  - "turborepo-tests/integration"
+  - "turborepo-tests/helpers"
+  - "turborepo-tests/examples/*"
```

Comment:
Why did the layout need to change? Can we keep each of the example tests in the top level?

Reply:
tomato tomato. I found it "cleaner" to not have an extra layer of nesting of workspaces. Go head and do what you prefer 👍🏾

Reply:
(also unnecessary for this PR and increases the diff to understand, fwiw)